### ISSUE TITLE
Importing a cylider fro BlenderFDS

### ISSUE THREAD

**Issue by fidran** [firemodels/fds#11348]
I have imported a cylinder from BlenderFDS as a bingeom file and added it to simple FDS case. I can move and tilt the cylinder, but when I add SURF_ID=’FIRE’ to the &GEOM line, I got an error message :

ERROR: problem with GEOM Cylinder, number of surfaces in SURF_ID field (1) not equal to number of surfaces (0) defined in bingeom Cylinder.bingeom. (CHID: SMT)

If I set &MISC SURF_DEFAULT =’FIRE’, it works.
Revision used : FDS6.7.9-0-gec52dee42-release

![Cylinder](https://user-images.githubusercontent.com/40716592/212781245-d2e41092-9593-408f-a2c6-7abbb0c44813.JPG)
[fds and bingeom.zip](https://github.com/firemodels/fds/files/10430039/fds.and.bingeom.zip)

**Comment by marcosvanella** [comment]
Define a 'FIRE' surf line and assign it's SURF_ID to the cylinder in Blenderfds. Otherwise the surface pointer for each cylinder triangle will point to the default surface.On Jan 16, 2023 6:55 PM, fidran ***@***.***> wrote:

I have imported a cylinder from BlenderFDS as a bingeom file and added it to simple FDS case. I can move and tilt the cylinder, but when I add SURF_ID=’FIRE’ to the &GEOM line, I got an error message :

ERROR: problem with GEOM Cylinder, number of surfaces in SURF_ID field (1) not equal to number of surfaces (0) defined in bingeom Cylinder.bingeom. (CHID: SMT)

If I set &MISC SURF_DEFAULT =’FIRE’, it works.

Revision used : FDS6.7.9-0-gec52dee42-release



fds and bingeom.zip
